api/application/client.go

```diff
@@ -772,3 +772,33 @@ func (c *Client) UnsetApplicationConfig(application string, options []string) er
 	}
 	return results.OneError()
 }
+
+// ResolveUnitErrors clears errors on one or more units.
+// Either specify one or more units, or all.
+func (c *Client) ResolveUnitErrors(units []string, all, retry bool) error {
+	// convert unit name to a tag
+	if len(units) > 0 && all == true {
+		return errors.New("provide units or '--all' but not both")
```

Comment:
The API doesn't know about flags. It shouldn't be saying '--all'.

Reply:
cut and paste error from original branch, fixed